embetter/external/__init__.py

```diff
@@ -1,14 +1,15 @@
 from embetter.error import NotInstalled
 
 try:
-    from ._openai import OpenAIEncoder
+    from ._openai import AzureOpenAIEncoder, OpenAIEncoder
```

Comment:
Would it not be better to split this import? 

If somebody wants to use the `OpenAIEncoder` now, they must also have the `AzureOpenAIEncoder` around, or get the not installed error.

Reply:
Maybe now, but it feels safer to assume that they might differ in the future.